src/node_report.cc

```diff
@@ -106,6 +106,8 @@ static char report_directory[NR_MAXPATH + 1] = ""; // defaults to current workin
 std::string version_string = UNKNOWN_NODEVERSION_STRING;
 std::string commandline_string = "";
 static TIME_TYPE loadtime_tm_struct; // module load time
+static time_t load_time; // module load time absolute
+static time_t current_time; // current time absolute
```

Comment:
I don't think we need either of these:
- load_time can be obtained by doing: `time_t load_time = mktime(&loadtime_tm_struct);`
- there's no point storing current_time in a static variable, we can't re-use it the next time node-report is invoked it'll be wrong. You can move `time(&load_time)` into `PrintResourceUsage`.